CMakeLists.txt

```diff
@@ -2,95 +2,100 @@
-
+	add_subdirectory(11_FFT)
 
 	# Waiting for a refactor
-	#add_subdirectory(27_PLYSTLDemo EXCLUDE_FROM_ALL)
-	#add_subdirectory(33_Draw3DLine EXCLUDE_FROM_ALL)
+	#add_subdirectory(27_PLYSTLDemo)
+	#add_subdirectory(33_Draw3DLine)
 
 	# Unit Test Examples
-	add_subdirectory(20_AllocatorTest EXCLUDE_FROM_ALL)
-	add_subdirectory(21_LRUCacheUnitTest EXCLUDE_FROM_ALL)
-	add_subdirectory(22_CppCompat EXCLUDE_FROM_ALL)
-	add_subdirectory(23_Arithmetic2UnitTest EXCLUDE_FROM_ALL)
-	add_subdirectory(24_ColorSpaceTest EXCLUDE_FROM_ALL)
+	add_subdirectory(20_AllocatorTest)
+	add_subdirectory(21_LRUCacheUnitTest)
+	add_subdirectory(22_CppCompat)
+	add_subdirectory(23_Arithmetic2UnitTest)
+	add_subdirectory(24_ColorSpaceTest)
 	add_subdirectory(25_FilterTest EXCLUDE_FROM_ALL)
-	add_subdirectory(26_Blur EXCLUDE_FROM_ALL)
-	add_subdirectory(27_MPMCScheduler EXCLUDE_FROM_ALL)	
-	add_subdirectory(28_FFTBloom EXCLUDE_FROM_ALL)
-	add_subdirectory(29_Arithmetic2Bench EXCLUDE_FROM_ALL)
-	# add_subdirectory(36_CUDAInterop EXCLUDE_FROM_ALL)
+	add_subdirectory(26_Blur)
+	add_subdirectory(27_MPMCScheduler)	
+	add_subdirectory(28_FFTBloom)
+	add_subdirectory(29_Arithmetic2Bench)
+	# add_subdirectory(36_CUDAInterop)
 
 	# Showcase compute pathtracing
-	add_subdirectory(30_ComputeShaderPathTracer EXCLUDE_FROM_ALL)
+	add_subdirectory(30_ComputeShaderPathTracer)
 
-	add_subdirectory(38_EXRSplit EXCLUDE_FROM_ALL)
+	add_subdirectory(38_EXRSplit)
 	# if (NBL_BUILD_MITSUBA_LOADER AND NBL_BUILD_OPTIX)
-	#	add_subdirectory(39_DenoiserTonemapper EXCLUDE_FROM_ALL)
+	#	add_subdirectory(39_DenoiserTonemapper)
 	# endif()
 
-	#add_subdirectory(43_SumAndCDFFilters EXCLUDE_FROM_ALL)
+	#add_subdirectory(43_SumAndCDFFilters)
 	add_subdirectory(47_DerivMapTest EXCLUDE_FROM_ALL)
 	add_subdirectory(54_Transformations EXCLUDE_FROM_ALL)
 	add_subdirectory(55_RGB18E7S3 EXCLUDE_FROM_ALL)
-	add_subdirectory(61_UI EXCLUDE_FROM_ALL)
-	add_subdirectory(62_CAD EXCLUDE_FROM_ALL)
+	add_subdirectory(61_UI EXCLUDE_FROM_ALL) # TODO: resurrect before `mesh_loaders` merge
+	add_subdirectory(62_CAD EXCLUDE_FROM_ALL) # TODO: Erfan, Przemek, Francisco and co. need to resurrect this
 	add_subdirectory(62_SchusslerTest EXCLUDE_FROM_ALL)
-	add_subdirectory(64_EmulatedFloatTest EXCLUDE_FROM_ALL)
+	add_subdirectory(64_EmulatedFloatTest)
 	add_subdirectory(0_ImportanceSamplingEnvMaps EXCLUDE_FROM_ALL) #TODO: integrate back into 42
 
 	add_subdirectory(66_HLSLBxDFTests EXCLUDE_FROM_ALL)
-	add_subdirectory(67_RayQueryGeometry EXCLUDE_FROM_ALL)
-	add_subdirectory(68_JpegLoading EXCLUDE_FROM_ALL)
+	add_subdirectory(67_RayQueryGeometry EXCLUDE_FROM_ALL) # TODO: resurrect before `mesh_loaders` merge
+	add_subdirectory(68_JpegLoading)
 
-  	add_subdirectory(70_FLIPFluids EXCLUDE_FROM_ALL)
-	add_subdirectory(71_RayTracingPipeline EXCLUDE_FROM_ALL)
+  	add_subdirectory(70_FLIPFluids)
+	add_subdirectory(71_RayTracingPipeline EXCLUDE_FROM_ALL) # TODO: resurrect before `mesh_loaders` merge
```

Comment:
@kevyuu in your PR with fixes for 67 and 71, you need to remove those `EXCLUDE_FROM_ALL`